android/build.gradle

```diff
@@ -5,19 +5,23 @@ buildscript {
 
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.3'
     }
 }
 
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion safeExtGet('compileSdkVersion', 26)
+    buildToolsVersion safeExtGet('compileSdkVersion', "26.0.3")
```

Comment:
This one needs to be `safeExtGet('buildToolsVersion', "26.0.3")`

Reply:
Gosh. I'm dumb. Yes. I'll fix that right now.